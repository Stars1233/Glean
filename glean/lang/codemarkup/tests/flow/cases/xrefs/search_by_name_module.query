query: flow.Module _
transform: [gensort, []]
