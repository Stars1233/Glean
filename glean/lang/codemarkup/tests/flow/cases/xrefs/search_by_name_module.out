[
  "@generated",
  { "key": { "file": { "key": "test/cjs_exports.js" } } },
  { "key": { "file": { "key": "test/es_exports.js" } } },
  { "key": { "file": { "key": "test/es_exports.js.flow" } } },
  { "key": { "file": { "key": "test/imports.js" } } },
  { "key": { "file": { "key": "test/subdir/more_cjs.js" } } },
  { "key": { "lib": "core.js" } },
  { "key": { "string_": "cjs_exports" } },
  { "key": { "string_": "react" } }
]